lms/djangoapps/ccx/tests/test_utils.py

```diff
@@ -62,7 +67,7 @@ def test_ccx_locator(self):
         self.assertEqual(result, ccx)
 
 
-class TestStaffOnCCX(CcxTestCase, SharedModuleStoreTestCase):
```

Comment:
Why did you remove SharedModuleStoreTestCase ? Doesn't that make the tests faster?